package-system/OpenSSL/test-linux.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+# The expected OPENSSL_VERSION_TEXT (Refer to build_config.json for the current version being built)
+EXPECTED_OPENSSL_VERSION="OpenSSL 1.1.1t  7 Feb 2023"
```

Comment:
The version actually has "  7 Feb 2023" in it?

Reply:
yes, that is the string based on the openssl_1.1.1t tag that is pulled